ui/pnpm-lock.yaml

```diff
@@ -4827,14 +4827,13 @@ packages:
     resolution: {integrity: sha512-bsTwuIg/BZZK/vreVTYYbSWoe2F+71P7K5QGEX+pT250DZbfU1MQ5prOKpPR+LL6uWKK3KMwMCAS74QB3Um1uw==}
     dependencies:
       browserslist: 4.21.7
-      caniuse-lite: 1.0.30001492
+      caniuse-lite: 1.0.30001597
       lodash.memoize: 4.1.2
       lodash.uniq: 4.5.0
     dev: true
 
-  /caniuse-lite@1.0.30001492:
-    resolution: {integrity: sha512-2efF8SAZwgAX1FJr87KWhvuJxnGJKOnctQa8xLOskAXNXq8oiuqgl6u1kk3fFpsp3GgvzlRjiK1sl63hNtFADw==}
-    dev: true
+  /caniuse-lite@1.0.30001597:
+    resolution: {integrity: sha512-7LjJvmQU6Sj7bL0j5b5WY/3n7utXUJvAe1lxhsHDbLmwX9mdL86Yjtr+5SRCyf8qME4M7pU2hswj0FpyBVCv9w==}
```

Comment:
@elmjag, we lost the `dev: true` in the operation, though.

I do not know if it has any meaningful consequence.

Maybe @axelboc knows.

Reply:
Hmm well spotted. Not a big deal I reckon, but just in case:

```
pnpm install --save-dev caniuse-lite@latest
```

Then remove `caniuse-lite` from `package.json` as you've done before and run `pnpm install` again to make sure it doesn't remodify `pnpm-lock.yaml`.